### ISSUE TITLE
cmsDriver does not give proper error message for missing --conditions option

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#12708]
@mtosi noticed that if one does not give `--conditions` to cmsDriver (in 80X), one gets

```
> cmsDriver.py tracking -s RAW2DIGI,RECO --no_exec
Traceback (most recent call last):
  File ".../CMSSW_8_0_X_2015-12-06-2300/bin/slc6_amd64_gcc493/cmsDriver.py", line 55, in <module>
    run()
  File ".../CMSSW_8_0_X_2015-12-06-2300/bin/slc6_amd64_gcc493/cmsDriver.py", line 11, in run
    options = OptionsFromCommandLine()
  File ".../CMSSW_8_0_X_2015-12-07-1100/python/Configuration/Applications/cmsDriverOptions.py", line 33, in OptionsFromCommandLine
    options=OptionsFromItems(sys.argv[1:])
  File .../CMSSW_8_0_X_2015-12-07-1100/python/Configuration/Applications/cmsDriverOptions.py", line 53, in OptionsFromItems
    possible+="\nauto:"+k+" -> "+autoCond.autoCond[k]
TypeError: cannot concatenate 'str' and 'tuple' objects
```

instead of the intended exception. I'm not sure where this should be reported, but since autoCond is part of this, I try AlCa first. @diguida @mmusich @franzoni

**Comment by mmusich** [comment]
addding PdmV @boudoul, @fabozzi, @srimanob, @hengne,

**Comment by Dr15Jones** [comment]
assign operations

**Comment by makortel** [comment]
Decided to fix myself, see #21648.
